src/main/kotlin/models/glTF/Primitive.kt

```diff
@@ -0,0 +1,8 @@
+package models.glTF
+
+import com.fasterxml.jackson.annotation.JsonInclude
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+class Primitive(val attributes: Attributes, val material: Int?) {
+    val mode = 4
```

Comment:
Could do with an explanation of why 4 (and/or define this as a constant or enum value somewhere)